openmc/material.py

```diff
@@ -1079,6 +1079,13 @@ def mix_materials(cls, materials, fracs, percent_type='ao', name=None):
         new_density = np.sum([dens for dens in mass_per_cc.values()])
         new_mat.set_density('g/cm3', new_density)
 
+        # If any of the involved materials is depletable, the new material is 
+        # depletable
+        for mat in materials:
+            if mat.depletable:
+                new_mat.depletable = True
+                break
```

Comment:
Python's `any` function would come in handy here:
```suggestion
        new_mat.depletable = any(mat.depletable for mat in materials)
```

Reply:
@paulromano thanks for the comment!